src/react/databrowser/objects/ObjectList.tsx

```diff
@@ -45,13 +45,16 @@ export default function ObjectList({
   const dispatch = useDispatch();
   const { pathname } = useLocation();
   const query = useQueryParams();
+  const searchInput = query.get('metadatasearch');
+
   const errorZenkoMsg = useSelector(
     (state: AppState) => state.zenko.error.message,
   );
   const isBucketVersioningDisabled = isVersioningDisabled(
     bucketInfo.versioning,
   );
-  const isMetadataType = listType === LIST_OBJECTS_METADATA_TYPE;
+
+  const isMetadataType = listType === LIST_OBJECTS_METADATA_TYPE || searchInput;
```

Comment:
Nit: It likely could be reduced to
```suggestion
  const isMetadataType = !!searchInput;
```
And we could drop the LIST_OBJECTS_METADATA_TYPE logic

Reply:
This can be done in a second iteration